### PR TITLE
feat: New login page style

### PR DESCRIPTION
**Pull request by SilviaTrujillano23**
## Issue ticket number and link
https://trello.com/c/naf9ozbW

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

## Pull request type

Please check the type of change your PR introduces:

- [x] :sparkles: Feature
- [ ] :bug: Bugfix
- [ ] :pencil2: Fixing typos
- [ ] :lipstick: :art: Code style update (formatting, renaming)
- [ ] :recycle: Refactoring (no functional changes, no api changes)
- [ ] :zap: Performance
- [ ] :pencil: Documentation content changes
- [ ] :white_check_mark: Unit testing
- [ ] :ok_hand: Updating due to code review suggest
- [ ] :rocket: Release: v
- [ ] :wrench: Add or update configuration files
- [ ] :hammer: Add or update development scripts
- [ ] :bricks: Infrastructure related changes

## What is the current module?

- Login page

## What is the new behavior?

- New style in login page
- Add icons in form inputs
- Add show/hide password 
- Render app bar in pages different from route /login
- Test should not render appbar in route login
- Mobile first

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

Mobile version
Screenshot 2023-10-13 at 00 56 17

Web version
Screenshot 2023-10-13 at 00 56 54

### CODE REVIEW

File: src/components/Main/Main.tsx [hunk 1]
```diff
@@ -17,28 +18,27 @@ export interface MainProps {
...
-      {matchesDesktopDisplay ? (
-        <AppBarWeb
-          version={config.version || ''}
-        />
+      {shouldShowAppBars && matchesDesktopDisplay ? (
+        <AppBarWeb version={config.version || ''} />
       ) : (
-        <AppBarMobile version={version} />
+        shouldShowAppBars && <AppBarMobile version={version} />
       )}
```
Comment (jlora-mck) [inline src/components/Main/Main.tsx:34]: Lets build a function to render this, note that the conditions are getting complex here and if in the future we want add more the JSX will need to get dirty

File: src/shared/styles/login.ts [hunk 2]
```diff
@@ -1,60 +1,77 @@
...
-      left: '30%',
-      width: '40%',
-      minHeight: '35%',
-      maxHeight: '35%',
-      [theme.breakpoints.down('sm')]: {
-        top: '20%',
-        left: 0,
-        width: '100%',
+      marginBottom: '250px',
+      [theme.breakpoints.up('sm')]: {
```
Comment (jlora-mck) [inline src/shared/styles/login.ts:8]: Remember that we agreed to build things as mobile first

**Review by jlora-mck — CHANGES_REQUESTED** [review]
Left a couple changes, Good work

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

File: src/components/Main/Main.spec.tsx [hunk 3]
```diff
@@ -32,4 +33,20 @@ describe('Main component', () => {
     const webAppBarVersion = screen.getByText(/v14/)
     expect(webAppBarVersion).toBeInTheDocument()
   })
+
+  xit('should not render AppBar components on /login route', () => {
```
[the remark this answers is not in the document]
Comment (SilviaTrujillano23) [reply src/components/Main/Main.spec.tsx:37]: Return test, runs fine.

File: src/components/Main/Main.tsx (hunk 1 above)
Comment (SilviaTrujillano23) [reply src/components/Main/Main.tsx]: done

File: src/shared/styles/login.ts (hunk 2 above)
Comment (SilviaTrujillano23) [reply src/shared/styles/login.ts:8]: Resolved, please let double check together after the demo

**Review by jlora-mck — APPROVED** [review]

**Pull request merged**